### PR TITLE
[SYCLomatic #1719] Enable CMake script migration for llama.cpp

### PR DESCRIPTION
**Pull request by tomflinda**
Signed-off-by: chenwei.sun <chenwei.sun@intel.com>

### CODE REVIEW

File: applications/config/TEMPLATE_llama.xml [hunk 1]
```diff
@@ -10,4 +10,4 @@
         <platformRule OSFamily="Windows" runOnThisPlatform="false" />
         <optlevelRule excludeOptlevelNameString="usmnone" />
     </rules>
-</test>
\ No newline at end of file
+</test>
```
Comment (intwanghao) [inline applications/config/TEMPLATE_llama.xml]: No change here.

File: applications/applications.xml [hunk 2]
```diff
@@ -4,5 +4,6 @@
     <description>applications test</description>
     <tests>
         <test testName="llama" configFile="config/TEMPLATE_llama.xml" />
+        <test testName="llama_cmake_migration" configFile="config/TEMPLATE_llama_cmake_migration.xml" />
```
Comment (intwanghao) [inline applications/applications.xml:7]: can reuse TEMPLATE_llama.xml

Comment (tomflinda) [reply applications/applications.xml:7]: cannot reuse TEMPLATE_llama.xml, as oneAPI env is only available for cpu device, so we need litmit the test case on cpu device.

File: applications/config/TEMPLATE_llama.xml (hunk 1 above)
Comment (tomflinda) [reply applications/config/TEMPLATE_llama.xml]: done

**Review by zhiweij1 — APPROVED** [review]

**Review by zhiweij1 — APPROVED** [review]

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**